jobs/base_company_sample.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+import string
+
+# [START instantiate]
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+client_service = build('jobs', 'v2')
+
+# [END instantiate]
+
+
+# [START basic_company]
```

Comment:
Maybe it's not necessary to declare a function here with a specific region tag. How about just specifying a `company` variable? 

```
 company = {
       'display_name': 'Google',
       'distributor_company_id': distributor_company_id,
       'hq_location': '1600 Amphitheatre Parkway Mountain View, CA 94043'
}
```

You can use a fixed ID here as well I think.

Reply:
I could not use a fixed ID since creating an entity with same ID would lead to the failure of tests

Reply:
No. This function is required from doc site and I couldn't use the fixed ID due to our system.